demo/demoproject/storage/tests.py

```diff
@@ -31,6 +32,30 @@ def test_download_response(self):
                                  basename='1.txt',
                                  mime_type='text/plain')
 
+    @temporary_media_root()
+    def test_not_modified_download_response(self):
```

Comment:
Rename the test to explain what it does as a spec, something like:
``test_download_response_returns_an_http_not_modified_response_if_unmodified(self)``